Utilities/build-script-helper.py

```diff
@@ -152,7 +150,7 @@ def add_common_args(parser):
     parser.add_argument('--cross-compile-host', help='cross-compile for another host instead')
     parser.add_argument('--cross-compile-config', help='an SPM JSON destination file containing Swift cross-compilation flags')
 
-  subparsers = parser.add_subparsers(title='subcommands', dest='action', metavar='action')
+  subparsers = parser.add_subparsers(title='subcommands', dest='action', required=True, metavar='action')
```

Comment:
Does this need the same treatment you gave the indexstore-db change to check the version?

Reply:
Yes, of course. Why did I forget it 🤦

Added it now.